receptacle.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.homepage = "https://github.com/andreaseger/receptacle"
   spec.license = "MIT"
 
-  spec.required_ruby_version = ">= 2.4"
```

Comment:
can you please make this 2.6 compatible? Need it for some specs to still run with this version of the receptacle gem on jruby

Reply:
additionally i want to upgrade the jruby version of workouts also to this version for a fair comparison

Reply:
Sorry, I tried to work on Ruby 2.6 support but it is making the code too complicated. I read that there isn't much difference in performance between ruby 2.6 vs 2.7 anyways https://engineering.appfolio.com/appfolio-engineering/2019/12/27/ruby-270s-rails-ruby-bench-speed-is-unchanged-from-260

Reply:
:( ok then i guess we can also work without it. jruby 9.3. is only 2.6 compatible unfortunately